syntax = "proto3";

package selector;

service ML{
    rpc PingOne(SelectorPing) returns (Empty);
    rpc PingAll(Empty) returns (Empty);
}

message SelectorPing{
    string user = 1;
    enum source{
        VK = 0;
        TG = 1;
    }
}

message Empty{}